kong/pdk/table.lua

```diff
@@ -53,7 +53,7 @@ local clear_tab = require "table.clear"
 local merge_tab = table_merge
 
 
-local function new(self)
+local function new(pdk)
```

Comment:
What about moving the changes from `self->pdk` to a separate PR?

Reply:
@jschmid1 the self->pdk commit was removed from this PR.